app/globals.css

```diff
@@ -16,18 +16,75 @@
   }
 }
 
-body {
+/* body {
   color: rgb(var(--foreground-rgb));
   background: linear-gradient(
       to bottom,
       transparent,
       rgb(var(--background-end-rgb))
     )
     rgb(var(--background-start-rgb));
+} */
```

Comment:
Wasn't sure what all of this was doing exactly, so replaced it with lines 29-33 below